getNodeByUnit/function.json

```
@@ -0,0 +1,18 @@
+{
+  "bindings": [
+    {
+      "authLevel": "anonymous",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "getNodeByUnit",
+      "methods": ["get"],
+      "route": "node/"
+    },
+    {
+      "type": "http",
+      "direction": "out",
+      "name": "$return"
+    }
+  ],
+  "scriptFile": "../dist/getNodeByUnit/index.js"
```

```suggestion
  "scriptFile": "../dist/GetNodeByUnit/index.js"
```
Change directory's name as well like the previous.

---

You need to change folders names as well (you can see that in the git files the name is not updated)